modules/ROOT/pages/indexes-for-nearest-neighbor.adoc

```diff
@@ -0,0 +1,431 @@
+:description: This chapter describes how to use vector indexes to perform approximate nearest neighbor search.
+
+:fn-hnsw: footnote:hnsw[See: http://dx.doi.org/10.1109/TPAMI.2018.2889473[Efficient and Robust Approximate Nearest Neighbor Search Using Hierarchical Navigable Small World Graphs] -- Yury A. Malkov and Dmitry A. Yashunin (preprint: https://arxiv.org/abs/1603.09320:[arXiv:1603.09320])]
+:fn-ieee-754: footnote:ieee-754[https://doi.org/10.1109/IEEESTD.2019.8766229[IEEE Standard for Floating-Point Arithmetic]]
+
+:link-operations-manual: link:{neo4j-docs-base-uri}/operations-manual/{page-version}
+:link-procedures-reference: {link-operations-manual}/reference/procedures/
+
+:ieee-754: IEEE 754{fn-ieee-754}
+
+:l2-norm: image:l2.svg["l2"]-norm
+
+[[indexes-vector]]
+= Nearest neighbor search
+
+[abstract]
+--
+This chapter describes how to use vector indexes to perform an approximate nearest neighbor search.
+--
+
+_This feature is introduced in Neo4j 5.11._ label:beta[]
+
+Vector indexes are powered by the link:https://lucene.apache.org/[Apache Lucene] indexing and search library, and can be used to index nodes by `LIST<FLOAT>` properties valid to the index.
```

Comment:
I know the full-text index page currently starts like this (I have not yet had the time to fix it), but I think it is not the most helpful way to start. Maybe moving forward, we could add an introduction section (including answers to questions such as "what are vector indexes?", "why would you use them?") to start the page with, to ease the learning curve for those not already initiated.